open default.yaml:  no such file or directory

### Checklist
- [X] Bug from Service (Vatz)
- [ ] Bug from Plugin (Vatz) 
- [ ] Malfunction on CI/CD 
- [ ] etc(e.g, Hardware version doesn't match, )

---

### **Describe the bug**
> A clear and concise description of what the bug is.

VATS start command returns panic if a default config file is not exist.

### **How to Reproduce**
> Steps to reproduce the behavior:

1. Execute `./vats start`, notice that `--config` parameter should be ommited.

### **Expected behavior**
> A clear and concise description of what you expected to happen.


```
panic: open default.yaml: no such file or directory

goroutine 1 [running]:
github.com/dsrvlabs/vatz/manager/config.InitConfig.func1()
        /home/rootwarp/git/vatz/manager/config/config.go:164 +0x166
sync.(*Once).doSlow(0xc000132a80?, 0xc000137e98?)
        /home/rootwarp/go1.18.1/src/sync/once.go:68 +0xc2
sync.(*Once).Do(...)
        /home/rootwarp/go1.18.1/src/sync/once.go:59
github.com/dsrvlabs/vatz/manager/config.InitConfig({0xaa4ded?, 0xaa3c00?})
        /home/rootwarp/git/vatz/manager/config/config.go:156 +0x58
main.createStartCommand.func2(0xc000232a00?, {0xa9ea93?, 0x0?, 0x0?})
        /home/rootwarp/git/vatz/cmd.go:122 +0x17e
github.com/spf13/cobra.(*Command).execute(0xc000232a00, {0xf8a3d0, 0x0, 0x0})
        /home/rootwarp/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:872 +0x694
github.com/spf13/cobra.(*Command).ExecuteC(0xc000232500)
        /home/rootwarp/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:990 +0x3b4
github.com/spf13/cobra.(*Command).Execute(...)
        /home/rootwarp/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:918
main.main()
        /home/rootwarp/git/vatz/main.go:50 +0xb4
```


### **Screenshots**
> If applicable, add screenshots to help explain your problem.

--- a/manager/config/config.go
+++ b/manager/config/config.go
@@ -136,7 +136,7 @@ func InitConfig(configFile string) *Config {
 		if err != nil {
 			if strings.Contains(err.Error(), "no such file or directory") {
 				log.Error().Str("module", "config").Msgf("loadConfig Error: %s", err)
-				log.Error().Str("module", "config").Msg("Please, initialize VATZ with command `.vatz init` to create config file `default.yaml` first or set appropriate path for config file `default.yaml`.")
+				log.Error().Str("module", "config").Msg("Please, initialize VATZ with command `./vatz init` to create config file `default.yaml` first or set appropriate path for config file `default.yaml`.")
 				os.Exit(1)
 			}
 			panic(err)

